### PR TITLE
Updated medium material icon demo

### PR DESCRIPTION
**Pull request by JoseAlba**
#### Don't forget:
- [x] Are you sure you mean this repo? This repository is a reference to the [main Flutter repository](https://github.com/flutter/flutter). Code-level issues should be created there.
- [ ] Link to GitHub issues it solves. ```closes #1234```
- [x] Sign the CLA bot. You can do this once the pull request is opened.

Be sure to read the [Flutter guide on contributing](https://github.com/flutter/flutter/blob/master/CONTRIBUTING.md).

### Related issues
 - Type `#` and the issue number or `#` and select the from the inline drop down.

### CODE REVIEW

**Review by guidezpl — APPROVED** [review]
```
  _    ___ _____ __  __ 
 | |  / __|_   _|  \/  |
 | |_| (_ | | | | |\/| |
 |____\___| |_| |_|  |_|
                        
```

File: medium/material_icons/main.dart [hunk 1]
```diff
@@ -17,26 +17,78 @@ class MyHomePage extends StatelessWidget {
...
+          crossAxisAlignment: CrossAxisAlignment.center,
+          children: [
+            Row(
+              mainAxisAlignment: MainAxisAlignment.spaceAround,
+              children: const [
+                Icon(
+                  Icons.wash_rounded,
+                  color: Colors.pink,
+                  size: 48.0,
+                  semanticLabel: 'Wash your hands',
```
Comment (rami-a) [inline medium/material_icons/main.dart:31]: Extract these strings to a final/const

File: medium/material_icons/main.dart [hunk 2]
```diff
@@ -17,26 +17,78 @@ class MyHomePage extends StatelessWidget {
...
-              semanticLabel: 'Wear a mask',
+          crossAxisAlignment: CrossAxisAlignment.center,
+          children: [
+            Row(
+              mainAxisAlignment: MainAxisAlignment.spaceAround,
+              children: const [
+                Icon(
+                  Icons.wash_rounded,
+                  color: Colors.pink,
+                  size: 48.0,
```
Comment (rami-a) [inline medium/material_icons/main.dart:30]: Make this size a const

**Review by rami-a — APPROVED** [review]

File: medium/material_icons/main.dart (hunk 1 above)
Comment (JoseAlba) [reply medium/material_icons/main.dart:31]: done

Comment (JoseAlba) [reply medium/material_icons/main.dart:31]: Done

File: medium/material_icons/main.dart (hunk 2 above)
Comment (JoseAlba) [reply medium/material_icons/main.dart:30]: Done

**Pull request merged**